# 
# @brief    : VISION_POSITION
# @time     : 2020/10/04
# @auhtor   : liuxu <liuxu.ccc@gmail.com> 
#

# position gotten from vision

geometry_msgs/Point car
geometry_msgs/Point board
geometry_msgs/Point head
int8 boardNum